state-chain/runtime/src/lib.rs

```diff
@@ -1039,8 +1038,83 @@ type AllMigrations = (
 	FlipToBurnMigration,
```

Comment:
@dandanlen The `pallet_cf_pools` migrations are commented out but not the `FlipToBurnMigration`, seems strange. I was unable to get my try-migrations to pass without uncommenting the pools pallet migrations. Is this expected?

Reply:
Yeah I agree, the pools migrations should not be commented. I'll fix it. 

I want to make some other small improvements.